# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../gmrun/gmrun-0.9.2-glibc210.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

http://sources.gentoo.org/cgi-bin/viewvc.cgi/gentoo-x86/x11-misc/gmrun/files/gmrun-0.9.2-glibc210.patch?revision=1.1
--- gmrun-0.9.2/src/gtkcompletionline.cc.orig	2009-06-19 22:38:14.000000000 +0400
+++ gmrun-0.9.2/src/gtkcompletionline.cc	2009-06-19 22:37:14.000000000 +0400
@@ -376,7 +377,7 @@
   return 0;
 }
 
-int my_alphasort(const void* va, const void* vb) {
+int my_alphasort(const dirent** va, const dirent** vb) {
   const struct dirent** a = (const struct dirent**)va;
   const struct dirent** b = (const struct dirent**)vb;
 
